distributions/application/src/main/java/com/connexta/ion/replication/application/Main.java

```diff
@@ -37,12 +36,7 @@
   public static void main(String[] args) throws Exception {
     // Need to load some system properties before anything else is done
     loadSslSystemProperties();
-    SpringApplication application = new SpringApplication(Main.class);
-    application.setWebApplicationType(WebApplicationType.NONE);
-    application.run(args);
-    while (true) {
```

Comment:
This is no longer needed. I believe it's because a tomcat server is now being spun up for exposing metrics and keeping the main thread busy.

Reply:
Disabling the replication metrics would mean only those metrics would not be reported. Metrics still exposed through the application.yml would still be reported. That is a good question though if someone disabled all metrics entirely, then no web server would be started.

Reply:
My point was that if metrics are disabled, won't the API provided also be disabled which means we would get back to having a spring app that exits right away?

Reply:
Just tested this out to make sure, and the tomcat server was still started up even though the endpoints were not exposed. This is because we still get the embedded tomcat server brought in by the spring-boot-starter-web dependency.